index.html

```diff
@@ -7,27 +7,44 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link href="http://fonts.cdnfonts.com/css/segoe-ui-4" rel="stylesheet">
     <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="css/style.css">
     <title>Blue Recycling</title>
     <link rel="stylesheet" href="styling.css">
 </head>
 
 <body>
-    <div id="navBar">
-        <img class="logoNav" src="logga-cropped.svg" alt="logo of Blue Recycling">
-        <nav>
-            <div id="burger">
-                <span></span>
-                <span></span>
-                <span></span>
+    <div class="landingPage">
+        <div id="navBar">
+            <img class="logoNav" src="logga-cropped.svg" alt="logo of Blue Recycling">
```

Comment:
Good with the alt text, but maybe it could describe the logo?

Also consider renaming the file to, for example, `blue-recycling-logo.svg` (improves SEO).